src/components/ebay-checkbox/index.js

```diff
@@ -7,5 +7,15 @@ module.exports = require('marko-widgets').defineComponent({
             value: originalEvent.target.value,
             checked: originalEvent.target.checked
         });
-    }
+    },
+    handleFocus: forwardEvent('focus')
 });
+
+function forwardEvent(eventName) {
```

Comment:
Either we should have handleChange call into this too (which I think might be the better way to do it), or we remove this and have handleFocus so the same as handleChange but with a different event.

Reply:
Yes, for consistency, let's have them both use this new `forwardEvent` function